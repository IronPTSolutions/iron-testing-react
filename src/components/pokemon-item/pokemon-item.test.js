import { render, screen } from "@testing-library/react";
import PokemonItem from "./pokemon-item";

describe("<PokemonItem/>", () => {
  test("happy case", () => {
    // Given
    const name = "Bulbasur";

    // When
    render(<PokemonItem name={name} />);

    // Then
    expect(screen.getByText(name)).toBeInTheDocument();
    expect(document.body).toMatchSnapshot();
  });
});
