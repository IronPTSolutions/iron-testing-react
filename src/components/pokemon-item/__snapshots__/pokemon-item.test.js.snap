// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PokemonItem/> happy case 1`] = `
<body>
  <div>
    <div>
      Bulbasur
    </div>
  </div>
</body>
`;
